apps/stops/lib/api.ex

```diff
@@ -420,5 +420,14 @@ defmodule Stops.Api do
 
   @spec get_zone_number(String.t() | nil) :: String.t() | nil
   defp get_zone_number(nil), do: nil
-  defp get_zone_number(zone), do: String.trim_leading(zone, "CR-zone-")
+
+  defp get_zone_number(zone) do
+    is_ferry_or_bus =
+      zone |> String.starts_with?("Boat-") or zone in ["ExpressBus-Downtown", "LocalBus"]
+
+    cond do
+      is_ferry_or_bus -> nil
+      true -> String.trim_leading(zone, "CR-zone-")
+    end
+  end
```

Comment:
Would it make sense to just check if it's a commuter rail since I think that's the only case we want the zones for? Then we're covered for whatever new modes or zones show up...

Maybe like (no idea if this actual works)

```elixir
case zone do
  "CR-zone-" <> zone_number -> zone_number
  _ -> nil
end
```